src/utils/timeUtils.ts

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { localize } from './localize';
+
+/**
+ * Get the elapsed time since `date` as a pretty printed string.
+ * @param date Date of event.
+ */
+export function timeDifferenceString(date: Date): string {
```

Comment:
> A lot of added code here is added to deal with displaying the times and date completed of the jobs and steps. We might try to rely on an npm package to do this for us, however I didn't immediately find one that suited our needs.

We use [moment](https://www.npmjs.com/package/moment) for this kind of thing. Specifically this: https://momentjs.com/docs/#/displaying/fromnow/

Reply:
Thanks! Now I can delete my code :)